scripts/generate-schemables.ts

```diff
@@ -28,6 +28,17 @@ type SchemableCombinators = {
   string: ReadonlyArray<readonly [string, string]>
 }
 
+type SchemableAugment = {
+  name: `With${string}`
+}
+
+const Augments: ReadonlyArray<SchemableAugment> = [
+  { name: 'WithBrand' },
+  { name: 'WithCheckDigit' },
+  { name: 'WithInvariant' },
+  { name: 'WithPattern' },
+]
```

Comment:
I thought about adding `WithRefine` and `WithUnknownContainers` in here but they added a lot of extra logic when it's easier to just leave them hard coded

Reply:
I ended up adding them in my branch, so no worries!  It was indeed a huge pain in the butt